File: iOSReview/iOSReview/ViewController.m
//
//  ViewController.m
//  iOSReview
//
//  Created by fishycx on 2020/12/7.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIView *outerView;

@property (weak, nonatomic) IBOutlet UIView *innerView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    CATransform3D outer = CATransform3DIdentity;
    outer.m34 = -1.0/500; //透视投影
    outer = CATransform3DMakeRotation(M_PI_4, 0, 1, 0);
    self.outerView.layer.transform = outer;
    
    CATransform3D inner = CATransform3DIdentity;
    inner.m34 = -1.0/500; //透视投影
    outer = CATransform3DMakeRotation(M_PI_4, 0, 0, 1);
    self.innerView.layer.transform = outer;
    
}


@end

File: leetcode/leetcode/ViewController.swift
//
//  ViewController.swift
//  leetcode
//
//  Created by 余传兴 on 2017/8/25.
//  Copyright © 2017年 fishycx. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var inputTextFiled: UITextField!
    @IBOutlet weak var result: UILabel!
    @IBOutlet weak var textField2: UITextField!
    @IBAction func submit(_ sender: Any) {
        self.result.text = "\(Solution().reverse(x: Int(inputTextFiled.text ?? "0")!))"
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.

    }
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}
